### PR TITLE
docs: add slash command builders example, fixes #7338

### LINKED ISSUE #7338: Examples link for builders and utilities don't exist
### Which package is this bug report for?

builders

### Issue description

Steps to reproduce:
1.Visit [NPM Package](https://www.npmjs.com/package/@discordjs/builders#Examples) or [Github](https://github.com/discordjs/discord.js/tree/main/packages/builders#Examples)
2.Click on [Slash Command Builders](https://github.com/discordjs/discord.js/blob/main/packages/builders/docs/examples/Slash%20Command%20Builders.md) Hyperlink found there

### Code sample

_No response_

### Package version

0.12.0

### Node.js version

v16.13.0

### Operating system

Not relevant

### Priority this issue should have

Medium (should be fixed soon)

### Which partials do you have configured?

Not applicable (subpackage bug)

### Which gateway intents are you subscribing to?

Not applicable (subpackage bug)

### I have tested this issue on a development release

_No response_

### PR DESCRIPTION
**Pull request by sanjaybaskaran01**
**Please describe the changes this PR makes and why it should be merged:**
Fixes #7338 and adds the example for slash command builder

**Status and versioning classification:**
Example taken from [here](https://github.com/discordjs/builders/blob/main/docs/examples/Slash%20Command%20Builders.md)

This PR **only** includes non-code changes, like changes to documentation, README

### CODE REVIEW

File: packages/builders/.gitignore [hunk 1]
```diff
@@ -17,9 +17,8 @@ pids
 # Dist
 dist/
 typings/
-docs/**/*
-!docs/index.yml
-!docs/README.md
```
Comment (iCrawl) [inline packages/builders/.gitignore:22]: ?

**Review by thebigwolfy — APPROVED** [review]

File: packages/builders/docs/examples/Slash Command Builders.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+		)
+		.addSubcommand((subcommand) =>
+			subcommand
+				.setName('user')
+				.setDescription("Lists a user's points")
+				.addUserOption((option) =>
+					option.setName('user').setDescription('The user whose points to list').setRequired(true),
+				),
+		),
+);
```
Comment (kyranet) [inline packages/builders/docs/examples/Slash Command Builders.md:98]:
```suggestion
const pointsCommand = new SlashCommandBuilder()
	.setName('points')
	.setDescription('Lists or manages user points')

	// Add a manage group
	.addSubcommandGroup((group) =>
		group
			.setName('manage')
			.setDescription('Shows or manages points in the server')
			.addSubcommand((subcommand) =>
				subcommand
					.setName('user_points')
					.setDescription("Alters a user's points")
					.addUserOption((option) =>
						option.setName('user').setDescription('The user whose points to alter').setRequired(true),
					)
					.addStringOption((option) =>
						option
							.setName('action')
							.setDescription('What action should be taken with the users points?')
							.addChoices([
								['Add points', 'add'],
								['Remove points', 'remove'],
								['Reset points', 'reset'],
							])
							.setRequired(true),
					)
					.addIntegerOption((option) => option.setName('points').setDescription('Points to add or remove')),
			),
	)

	// Add an information group
	.addSubcommandGroup((group) =>
		group
			.setName('info')
			.setDescription('Shows information about points in the guild')
			.addSubcommand((subcommand) =>
				subcommand.setName('total').setDescription('Tells you the total amount of points given in the guild'),
			)
			.addSubcommand((subcommand) =>
				subcommand
					.setName('user')
					.setDescription("Lists a user's points")
					.addUserOption((option) =>
						option.setName('user').setDescription('The user whose points to list').setRequired(true),
					),
			),
	);
```

File: packages/builders/docs/examples/Slash Command Builders.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+		.setName('boop_reminder')
+		.setDescription('How often should we remind you to boop the user')
+		.addChoice('Every day', 1)
+		.addChoice('Weekly', 7)
+		// Or, if you prefer adding more choices at once, you can use an array
+		.addChoices([
+			['Every three months', 90],
+			['Yearly', 365],
+		]),
+);
```
Comment (kyranet) [inline packages/builders/docs/examples/Slash Command Builders.md:42]:
```suggestion
const boopCommand = new SlashCommandBuilder()
	.setName('boop')
	.setDescription('Boops the specified user, as many times as you want')
	.addUserOption((option) => option.setName('user').setDescription('The user to boop').setRequired(true))

	// Adds an integer option
	.addIntegerOption((option) =>
		option.setName('boop_amount').setDescription('How many times should the user be booped (defaults to 1)'),
	)

	// Supports choices too!
	.addIntegerOption((option) =>
		option
			.setName('boop_reminder')
			.setDescription('How often should we remind you to boop the user')
			.addChoice('Every day', 1)
			.addChoice('Weekly', 7)
			// Or, if you prefer adding more choices at once, you can use an array
			.addChoices([
				['Every three months', 90],
				['Yearly', 365],
			]),
	);
```

**Review by kyranet — CHANGES_REQUESTED** [review]
Ran the examples thru prettier and also made some minor stylistic changes.

**Review by iCrawl — APPROVED** [review]

**Pull request merged**